_plugins/author.rb

```diff
@@ -66,6 +65,13 @@ def with_pic(input)
         "<img class='img-circle team-img bio-clip' src='/assets/images/18f.png' alt='18F logo'>"
       end
     end
+    def lookup(input, args)
+      args = args.split(',')
+      dataset = args[0].strip
+      key = args[1].strip unless args[1].nil?
```

Comment:
Should `args[1]` ever be `nil`? If so, that will throw off the `data[input][key]` lookup.